### PR TITLE
Add mumu emoji picker app

### PR DESCRIPTION
**Pull request by bval**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/mumu.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'mumu' do
+  version :latest
+  sha256 :no_check
+
+  # vendors.paddle.com was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/mumu.rb:5]: Insufficient to disambiguate.

File: Casks/mumu.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  url 'https://vendors.paddle.com/download/product/597910'
```
Comment (vitorgalvao) [inline Casks/mumu.rb:6]: Use the last redirect URL: `https://paddle.s3.amazonaws.com/fulfillment_downloads/116824/597910/3UjvmpBGSxuQuBAVlQm6_Mumu%201.0.1.dmg`

And use interpolation in the version.

File: Casks/mumu.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+cask 'mumu' do
+  version :latest
```
Comment (vitorgalvao) [inline Casks/mumu.rb:2]: Add a version.

File: Casks/mumu.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/mumu.rb:3]: Add a checksum

File: Casks/mumu.rb (hunk 2 above)
Comment (vitorgalvao) [reply Casks/mumu.rb:6]: Add an [`appcast`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md): `https://macupdater.net/cgi-bin/check_urls/check_url_redirect.cgi?url=https://vendors.paddle.com/download/product/597910`

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by bval** [comment]
🎉 Thanks, @ran-dall!

**Pull request merged**